config/routes.rb

```diff
@@ -4,10 +4,9 @@
                   contraints: {subdomain: 'api'},
                   paths: '/' do
     # /v1/
-    scope module: :v1 do
-      
+    scope module: :v1,
+          constraints: ApiConstraints.new(version: 1, default: true) do
```

Comment:
By using the constraints, we are forcing the values of the ApiConstraint to be version 1